patsy/__main__.py

```diff
@@ -75,25 +78,35 @@ def main() -> None:
         if result:
             sys.stderr.write(result)
             sys.stderr.write('\n\n')
+
     except DatabaseNotSetError:
-        sys.stderr.write('The "-d" argument was not set nor was the "PATSY_DATABASE" environment variable.\n')
+        logging.error('The "-d" argument was not set nor was the "PATSY_DATABASE" environment variable.')
         sys.exit(1)
+
     except OperationalError as e:
         error = str(e.orig)
-        sys.stderr.write(f'SQLAlchemy OperationalError: {error}\n')
+        logging.error(error)
         sys.exit(1)
+
     except InvalidStatusCodeError:
-        sys.stderr.write(
-            'An error occured when using the API. This could be due to the servers, '
-            'or the headers provided may be incorrect.\n'
+        logging.error(
+            'An error occured when using the API. This could be due to the servers, \
```

Comment:
This (and other string literals that are split across multiple lines) can be written to take advantage of the Python parser's automatic string concatenation (as it was with the `sys.stderr.write()` version.